docs/features/highlight.md

```diff
@@ -112,12 +112,12 @@ The {@link module:highlight/highlight~Highlight} plugin registers:
 	The `value` corresponds to the `model` property in configuration object. For the default configuration:
 	```js
 	highlight.options = [
-		{ model: 'yellowMarker', class: 'marker-yellow', title: 'Yellow Marker', color: '#fdfd77', type: 'marker' },
-		{ model: 'greenMarker', class: 'marker-green', title: 'Green marker', color: '#63f963', type: 'marker' },
-		{ model: 'pinkMarker', class: 'marker-pink', title: 'Pink marker', color: '#fc7999', type: 'marker' },
-		{ model: 'blueMarker', class: 'marker-blue', title: 'Blue marker', color: '#72cdfd', type: 'marker' },
-		{ model: 'redPen', class: 'pen-red', title: 'Red pen', color: '#e91313', type: 'pen' },
-		{ model: 'greenPen', class: 'pen-green', title: 'Green pen', color: '#118800', type: 'pen' }
+		{ model: 'yellowMarker', class: 'marker-yellow', title: 'Yellow Marker', color: 'var(--ck-marker-yellow)', type: 'marker' },
```

Comment:
There's not a single word in the documentation that the highlight feature comes with a style sheet, which defines a couple of markers and pens using CSS variables (a link to this [page](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables) will be necessary too, it's a new thing around the web). 

This must be clear, probably a new sub-section under **Configuring the highlight options**.

Reply:
You are right, I'll add another example with configuring other than default colors 👍